src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -289,6 +290,12 @@ const formConfig = {
           uiSchema: summaryOfDisabilities.uiSchema,
           schema: summaryOfDisabilities.schema,
         },
+        conclusionUnemployability: {
```

Comment:
Can we add a `depends` here for the `view:unemployabilityStatus` so that it doesn't always show up.

Reply:
done!